### PR TITLE
Add Mautic

### PR DESCRIPTION
**Pull request by RCheesley**
We would love to add Mautic to your site.

- available maintainers who are interested in guiding new contributors ✅ 
- the project curates a list of small tasks which are ideal for new contributors ✅ 
- maintainers are happy and willing to review and work with new contributors to help them learn more about open source ✅ 

Question: If we also wanted to add our documentation / dev docs, could we do that as well?  Would that mean a separate PR or can we include more than one repo?

### CODE REVIEW

File: _data/projects/mautic.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+desc: An open source marketing automation platform.
+site: https://www.mautic.org
+tags:
+  - mautic
+  - marketing
+  - marketing-automation
+  - php
+upforgrabs:
+  name: good first issue
+  link: https://github.com/mautic/mautic/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```
Comment (ritwik12) [inline _data/projects/mautic.yml]: @RCheesley Please change this to https://github.com/mautic/mautic/labels/good%20first%20issue

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/mautic.yml (hunk 1 above)
Comment (RCheesley) [reply _data/projects/mautic.yml]: Done - sorry just did a copy/paste of only open issues with the label applied, have updated as requested!

**Comment by RCheesley** [comment]
Seems that change has broken the check @ritwik12 - does the name also need hyphens, maybe? (does not have hyphens in our repo)

**Comment by RCheesley** [comment]
Please advise how to proceed given that the changes requested are now throwing an error @ritwik12

File: _data/projects/mautic.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: Mautic
+desc: An open source marketing automation platform.
+site: https://www.mautic.org
+tags:
+  - mautic
+  - marketing
+  - marketing-automation
+  - php
+upforgrabs:
+  name: good first issue
```
Comment (ritwik12) [inline _data/projects/mautic.yml:10]: Can you check if it is not having any extra space? Looks like this is throwing an error. @RCheesley

**Review by ritwik12 — DISMISSED** [review]

File: _data/projects/mautic.yml (hunk 2 above)
Comment (RCheesley) [reply _data/projects/mautic.yml:10]: Not that I can tell - where do you see there being an extra space?

It was passing until I changed the link to the one that you suggested - https://github.com/up-for-grabs/up-for-grabs.net/pull/3291#discussion_r931015543 which I changed here: https://github.com/up-for-grabs/up-for-grabs.net/pull/3291/commits/50ab06e252236932c79c865b77b675d00c239c1e. Thanks!

**Review by shiftkey — APPROVED** [review]
This appears to be working now, and I've manually verified the link works.

Thanks for the contribution @RCheesley!

**Pull request merged**